artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java

```diff
@@ -64,7 +64,7 @@
    @Message(id = 211001, value = "session created", format = Message.Format.MESSAGE_FORMAT)
    void dumpingSessionStack(@Cause Exception e);
 
-   @LogMessage(level = Logger.Level.INFO)
+   @LogMessage(level = Logger.Level.DEBUG)
```

Comment:
Client side this is very useful information, and should only occur on connection

Reply:
@michaelandrepearce ok, let me take a look

Reply:
Probably one option would be to make NettyConnector take a boolean value isServerConnection or something, then later 

      if (isServerConnection) {
         ActiveMQClientLogger.LOGGER.startedServerNettyConnector(connectorType,  TransportConstants.NETTY_VERSION, host, port);
      } else {
         ActiveMQClientLogger.LOGGER.startedNettyConnector(connectorType,  TransportConstants.NETTY_VERSION, host, port);
      }

Where the startedServerNettyConnector could log at DEBUG

Reply:
@michaelandrepearce, thanks, i am trying to do the same fix atm

Reply:
Agreed server side, but client side this is far too useful to lose.

Reply:
@michaelandrepearce actually this is getting too verbose on clients.
We can use log.debug on the client... and you can increase logger if you really want it.

Some users will complain the opposite.

Reply:
disagree still but as its 2 v 1 and its logging at the end of the day (bigger issues to really worry on) ill make it a -0. Ive merged the PR as is.